voskcli/transcribe.py

```diff
@@ -212,7 +212,70 @@ def transcribe(inputFile, outputFile, model):
           f'of {avg_confidence:.2f}. Saving WebVTT file...')
     vtt.save(outputFile)
     print('WebVTT saved.')
-    # print(vtt.content)
+
+
+def detect_model(inputFile, probeTime):
+    '''
+    Detect best model to use.
+
+    :param inputFile: Path to input file
+    :type inputFile: str
+    :param probeTime: Probe time in seconds
+    :type probeTime: int
+    :return: Best model to use
+    :type model: str
+    '''
+    available_models = glob('./models/*') + glob('/usr/share/vosk/models/*')
```

Comment:
> For a future feature, it could be nice to allow users to decide which models they want to use for auto-detection. What do you think about that?

You mean something like

```
vosk-cli --auto-detect --model model-de-0.1 --model model-en-0.23 …
```

Reply:
Right, something like that. I was imagining a use case were someone has many different language models in their `*/models` folder, but knows that they don't need them all for auto-detection. Whether there are uses cases where that is actually realistic I am not sure about though ^^'